cmd/mountlib/mount.go

```diff
@@ -283,11 +286,21 @@ func NewMountCommand(commandName string, hidden bool, mount MountFn) *cobra.Comm
 
 // Mount the remote at mountpoint
 func (m *MountPoint) Mount() (mountDaemon *os.Process, err error) {
-
 	// Ensure sensible defaults
 	m.SetVolumeName(m.MountOpt.VolumeName)
 	m.SetDeviceName(m.MountOpt.DeviceName)
 
+	// Handle the --remount flag by attempting to unmount the mountpoint first
+	if m.MountOpt.Remount {
```

Comment:
I'd probably put `if runtime.GOOS == linux && m.MountOpt.Remount {` here.

The compiler is smart enough to evaluate `runtime.GOOS == linux` at compile time then figure out it doesn't need to compile any of the stuff in the `if` statement so it saves bloating the binaries where this doesn't work.